lienzo-core/src/main/java/com/ait/lienzo/client/core/types/JsLienzoAnimations.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.types;
+
+import com.ait.lienzo.client.core.animation.AnimationProperties;
+import com.ait.lienzo.client.core.animation.AnimationProperty;
+import com.ait.lienzo.client.core.animation.AnimationTweener;
+import com.ait.lienzo.client.core.shape.IPrimitive;
+import com.ait.lienzo.client.core.shape.Shape;
+import jsinterop.annotations.JsType;
+
+// TODO: Stateless - Static methods?
+@JsType
+public class JsLienzoAnimations {
```

Comment:
Are we exposing this `JsLienzoAnimations` class into the public API, or is someone using it actually? This class was just a PoC, but I would prefer not adding this yet, if not necessary.

Reply:
Hi @romartin removed